modules/lang-mustache/src/test/java/org/elasticsearch/script/mustache/CustomMustacheFactoryTests.java

```diff
@@ -37,14 +37,22 @@
 public class CustomMustacheFactoryTests extends ESTestCase {
 
     public void testCreateEncoder() {
-        IllegalArgumentException e = expectThrows(IllegalArgumentException.class, () -> CustomMustacheFactory.createEncoder(null));
```

Comment:
This test is bogus. It can never happen in production code that `CustomMustacheFactory#createEncoders` is called with a `null` content type. The reason this test breaks with the change to use Java 11 collection conveniences is that all the immutable collections in Java 9 do not allow `null` entries and will throw NPE if you try to add a `null` entry to an immutable collection, or get a `null` key from an immutable map. That means the invocation here changes from throwing an `IllegalArgumentException` (because a `null` value is returned) to a `NullPointerException` because the Java 9 immutable collection rejects getting a `null` key. Since this, again, can not happen in production code, we remove the bogus test.